aces/Loris
# [Testing] Add PhanUndeclaredVariable rule to static analysis

### Brief summary of changes

Don't allow LORIS to proceed when using undeclared variables.

This PR contains a hybrid approach to fixing the issues. In some cases simple initializations were all that were needed. In others, I had to dig through old commits to find out what the variables should use.

There are some cases where I deleted old code when I couldn't imagine how it could work when the variable was not defined. This is what happened with the `bvl_feedback` ajax scripts. I assumed that this code simply isn't used generally and so only exists vestigially in LORIS. Please correct me if I've made a mistake.

### Related

Resolves https://github.com/aces/Loris/issues/4267

### To Test
On major, delete the phan suppression line `PhanUndeclaredVariable` from .phan/config.php. Run phan and you should get many errors.

Checkout my branch and run phan. The errors should be gone.
